src/create-storage.js

```diff
@@ -11,7 +11,7 @@ let isWorkerAvailable = window.Worker;
 
 let workify, worker;
 if (isWorkerAvailable) {
-	workify = require('webworkify');
+	workify = isWebpack ? require('webworkify-webpack') : require('webworkify');
```

Comment:
Yes, I mean `browser` field in `gl-waveform`'s `package.json`. But seems webpack takes it into account so simple replacing `webpack-webworkify` to `webworkify` won't work. I thought user can setup webpack-only replacement somehow the other way.